src/components/DelegationActionBar/DelegationActionBar.tsx

```diff
@@ -73,16 +72,13 @@ export const DelegationActionBar = ({
       <div className={classes.newApiAccordionContent}>
         {scopeList.length > 0 && (
           <div>
-            <p className={classes.scopeText}>{t('api_delegation.scopes')}:</p>
+            <h4 className={classes.scopeText}>{t('api_delegation.scopes')}:</h4>
             <ScopeList scopeList={scopeList} />
           </div>
         )}
         {topContentText && (
           <div>
-            <div className={classes.line}>
-              <Line />
-            </div>
-            <p className={classes.scopeText}>{t('api_delegation.description')}</p>
+            <h4 className={classes.scopeText}>{t('api_delegation.description')}</h4>
```

Comment:
This wasn't changed in this PR, but I just noticed this now:
Maybe change the classname scopeText if it is used on more elements than just scopes?